.env.sample

```diff
@@ -0,0 +1,2 @@
+JWT_SECRET=3EK6FD+o0+c7tzBNVfjpMkNDi2yARAAKzQlk8O2IKoxQu4nF7EdAh8s3TwpHwrdWT6R
```

Comment:
(comment): Make sure that the `.env` file in production does not have the same JWT key. That would be a major security vulnerability.